pkg/restapi/vc/operation/operations_test.go

```diff
@@ -90,8 +141,15 @@ const invalidVC = `{
   "issuanceDate": "2010-01-01T19:23:24Z"
 }`
 
+// ErrVaultNotFound
+var ErrVaultNotFound = errors.New("vault not found")
+
+// ErrDocumentNotFound
+var ErrDocumentNotFound = errors.New("edv does not have a document associated with ID")
```

Comment:
Should these be defined in the operation file? Then you can refer to them here and also there and avoid code duplication

Reply:
Right now i am using this for testing purposing only. They are not used in the operation file. May be they should not be exported.

Reply:
Ah ok. Now that they're unexported you can delete the comment entirely if you wish! Up to you :)